src/main/java/core/db/VereinTable.java

```diff
@@ -41,24 +43,21 @@ protected void initColumns() {
 	 * speichert das Verein
 	 */
 	void saveVerein(int hrfId, Verein verein) {
-		String statement = null;
+		String statement ;
 		final String[] awhereS = { "HRF_ID" };
 		final String[] awhereV = { "" + hrfId };
 
 		if (verein != null) {
-			//erst Vorhandene Aufstellung löschen
+			//first delete existing entry
 			delete( awhereS, awhereV );
 
-			//insert vorbereiten
-			statement = "INSERT INTO "+getTableName()+" ( TWTrainer , COTrainer , Physiologen , Pschyologen , Finanzberater , PRManager , Aerzte , Jugend , Siege , Ungeschlagen , Fans , TacticAssist , FormAssist, HRF_ID ) VALUES(";
+			//create sql insertion statement
+			statement = "INSERT INTO "+getTableName()+" ( COTrainer , Pschyologen , Finanzberater , PRManager , Aerzte , Jugend , Siege , " +
+					"Ungeschlagen , Fans , GlobalRanking , LeagueRanking , RegionRanking , PowerRating , TacticAssist , FormAssist, HRF_ID ) VALUES(";
 			statement
 				+= (""
```

Comment:
That first empty string could be removed too.

Reply:
which empty string ?